### DISCUSSION TITLE
`--code-font-family` 的值被 `--article-font-family` 覆盖

### DISCUSSION THREAD

**Discussion by DreamOneX** [CaiJimmy/hugo-theme-stack#747] [Q&A] [answered]
如题，在渲染`<code>sth</code>`时，`sth` 的字体是 `--article-font-family` 的值

所用浏览器:
* Mozilla Firefox 108.0.1
* Chromium 108.0.5359.124 Arch Linux

hugo 版本: `hugo v0.100.2-d25cb2943fd94ecf781412aeff9682d5dc62e284+extended`

stack 版本: `v3.13.0`

测试在 devtool 中修改 `--article-font-family`，`sth` 的字体会跟 **随其值改变而改变**

奇怪的是，devtool 中显示 `--code-font-family` **生效**，使用 devtool 修改亦无效

请问有什么解决方案 &.&

**Comment by DreamOneX** [comment]
试了下，在 `layouts/partials/head/custom.html` 中内联css指定 `--code-font-family` 是可行的，而只在 `variables.scss`  中修改 `--code-font-family` 则会被覆盖

(我怀疑与我在 `layouts/partials/head/custom.html` 修改了 `--article-font-family` 有关，所以我把它注释了，`variables.scss`  中的 `--code-font-family` 依旧被覆盖，但 devtools 显示它生效)

然而在 `variables.scss` 中修改颜色是可行的，这好像有点怪 :-)

**Comment by CaiJimmy** [comment, marked as answer]
感觉有点像缓存问题，我也没什么头绪... 

修改变量推荐在博客根目录创建 `assets/scss/custom.scss` 来重写，不推荐直接修改源文件。

**Reply by DreamOneX** [reply]
我说的修改是指把源文件拷贝到博客根目录对应位置修改（x

在 `assets/scss/custom.scss` 里重写可以 @_@
